### ISSUE TITLE
Bug - aioseop-preview-snippet.js console errors in Classic Editor

### ISSUE THREAD

**Issue by clutchwave** [semperfiwebdesign/all-in-one-seo-pack#3052]
![image](https://user-images.githubusercontent.com/2278974/69109704-2dbf8e00-0a3e-11ea-8446-10ce23dcde7a.png)

Please fill out this form as completely as possible so that we can help you more quickly.

- [x] I understand that this is a development repository and not a support forum. This is a bug report and is not a support request.
 - [ ] I've followed the [troubleshooting steps](https://semperplugins.com/documentation/how-to-troubleshoot-issues-with-all-in-one-seo-pack)  to determine if there's a theme/plugin conflict. 

**Provide the relevant URL(s) here**
`/wp-admin/post-new.php`
**Describe the bug**
```
Uncaught TypeError: Cannot read property 'on' of undefined
    at aioseop-preview-snippet.js:43
Uncaught TypeError: Cannot read property 'innerHTML' of undefined
    at aioseopUpdatePreviewSnippet (aioseop-preview-snippet.js:91)
    at aioseop-preview-snippet.js:72
Uncaught TypeError: Cannot read property 'on' of undefined
    at aioseop-count-chars.js:82
Uncaught TypeError: Cannot read property 'innerHTML' of undefined
    at aioseopUpdatePreviewSnippet (aioseop-preview-snippet.js:91)
    at HTMLInputElement.<anonymous> (aioseop-preview-snippet.js:66)
    at HTMLInputElement.dispatch (load-scripts.php?c=0&load[chunk_0]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery-ui-position,jquery-ui-mouse,moxiejs,plupload&ver=a56d5726606b690d3dd8ed8e8f94189c:3)
    at HTMLInputElement.r.handle (load-scripts.php?c=0&load[chunk_0]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery-ui-position,jquery-ui-mouse,moxiejs,plupload&ver=a56d5726606b690d3dd8ed8e8f94189c:3)
```

Now my Classic Editor is a bit messed up and isn't clearing out autopopulated gray text like "Add Title" -- not sure if

**To Reproduce**
1. Open the classic editor

**Expected behavior**
No console errors

**Screenshots**
![image](https://user-images.githubusercontent.com/2278974/69109704-2dbf8e00-0a3e-11ea-8446-10ce23dcde7a.png)

**Version Info (please complete the following information):**
 - WordPress version: 5.3
 - All in One SEO Pack version: 3.3.3
 - PHP version: php7.3-fpm

**Comment by arnaudbroes** [comment]
@clutchwave thank you for reporting this. We're not able to reproduce this using just the latest version of All in One SEO Pack and the Classic Editor. 

Please use these steps to check for a plugin/theme conflict and report back here - https://semperplugins.com/documentation/how-to-troubleshoot-issues-with-all-in-one-seo-pack/

**Comment by michaeltorbert** [comment]
@clutchwave I noticed you closed this issue. What was your finding?

**Comment by clutchwave** [comment]
Oh sorry, the errors still occur, but it's likely because of a plugin I'm using (Classic Editor). I don't use Gutenberg. Everything functions fine. The line with the issue:

```js
			setTimeout(function () {
				tinymce.editors[0].on('KeyUp', function () {
					aioseopUpdatePreviewSnippet();
				});
			}, 1000);
```

**Comment by michaeltorbert** [comment]
@clutchwave Can you provide steps to reproduce the problem?

**Comment by wpsmort** [comment]
@arnaudbroes I have now been able to reproduce this.  Here are the steps to reproduce:

1. Install the current version AIOSEOP on a site
2. Install and activate the Classic Editor plugin and set the default editor to Classic Editor
3. Go to Posts > Add New 
4. Switch to the Text tab 
5. Type some text in the Post Title field and click Save Draft
6. You should see the JS errors in the browser error console 

Note: This only happens when you have the Text tab as the active editor.  It does not happen with the Visual tab.

**Comment by arnaudbroes** [comment]
Just a status update: I'm working on this but this is going to take a bit of time. Unfortunately, the TinyMCE editor is not only badly documented but also most of the documentation doesn't apply to the specific implementation of TinyMCE within WordPress.

**Comment by wpsmort** [comment]
@clutchwave We have released a new version (v3.3.4) today that should fix this.

**Comment by clutchwave** [comment]
> We have released a new version (v3.3.4) today that should fix this.

Thanks!! Sorry I couldn't reproduce it. It would not always happen and I had too many plugins causing possible conflicts. And sadly, no true dev setup outside of production stuff.  Biggest thanks goes to @wpsmort
